include/boost/geometry/algorithms/detail/envelope/implementation.hpp

```diff
@@ -43,27 +46,31 @@ namespace detail { namespace envelope
 {
 
 
-struct exterior_ring_expand_policy
-{
-    template <typename Box, typename Geometry>
-    static inline void apply(Box& mbr, Geometry const& geometry)
-    {
-        Box ring_mbr;
-        envelope_range<>::apply(exterior_ring(geometry), ring_mbr);
-        geometry::expand(mbr, ring_mbr);
-    }
-};
-
-
+template <std::size_t Dimension, std::size_t DimensionCount>
```

Comment:
Why is the `envelope_polygon` dispatched with `Dimension`? Does it mean that this template may be instantiated for various Dimensions? If that was true then it would mean that the loop in `envelope_range` would be there in the code several times (one for each Dimension). In it was not true and always a specific Dimension was passed here then what's the purpose of those parameters?